### ISSUE TITLE
Remove AA/AAA pass/fail indication for colour deficiency results?

### ISSUE THREAD

**Issue by patrickhlauke** [ThePacielloGroup/CCAe#19]
Worth discussing/checking, but: it may be misleading to show pass/fail for the colour deficiency results. WCAG doesn't require you to test against those, so it's not correct to make pass/fail assessments based on simulated colour deficiency models. By all means keep the contrast ratio number indication, but those pass/fail indicators should likely be removed.

> ![capture](https://user-images.githubusercontent.com/895831/43829044-f4f8ee44-9af5-11e8-8ec9-7abb52037b37.PNG)

**Comment by ferllings** [comment]
I agree, Colour Deficiency is not included in the AA or AAA checkpoint, and the indicator should be removed.
